src/WebApi/appsettings.json

```diff
@@ -5,8 +5,12 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  // NOTE: If any of your connection strings real passwords, you should delete them and put them in the user secrets file instead.
```

Comment:
does json comments actually work as expected? i haven't dared use json comments in a lonnnnnnng time.

Locally vs deployed to an appservice shouldn't matter right?

Reply:
I'll give this a test.  Thanks for the suggestion.

Reply:
Gave this a go, and everything works fine.

https://deleteme-dev-app.azurewebsites.net/swagger

![image](https://github.com/SSWConsulting/SSW.CleanArchitecture/assets/2636640/a4e1eb1d-9423-4960-9471-8af5cfdc0047)

![image](https://github.com/SSWConsulting/SSW.CleanArchitecture/assets/2636640/bb8e77c0-a83b-4969-8462-761a552a90e7)